walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -270,5 +272,153 @@ public void notEnoughRoom() {
 
     Assert.assertEquals(expected, actual);
   }
+
+  @Test
+  public void optionalAttendeeUnavailable_returnsMandatoryAvailability() {
+    // Have each person have different events. An optional attendee C has an all day event. We should see 
+    // three options because only A & B are relevant, as C is not able to make any time frame. 
+    //
+    // Events  :       |--A--|     |--B--|
+    //           |-------------C---------------|
+    // Day     : |-----------------------------|
+    // Options : |--1--|     |--2--|     |--3--|
+
+    Collection<Event> events = Arrays.asList(
```

Comment:
ImmutableList.of(Event 1, Event 2, Event 3)

Prefer Immutable[Set|List].of(arg, arg, arg) over Arrays.asList

Reply:
adjusted